src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,38 @@
+.stars {
+  display: flex;
+  gap: 4px;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
```

Comment:
Ensure that the image path '../images/star.svg' is correct and that the image file exists at this location.